mlflow/ml-package-versions.yml

```diff
@@ -593,7 +593,7 @@ langchain:
       pip install git+https://github.com/langchain-ai/langchain#subdirectory=libs/langchain
   models:
     # Where the large package update was made (langchain-core, community, ...)
-    minimum: "0.2.16"
```

Comment:
In the next minor version update, `0.2.16` will be dropped, but we can drop it earlier. We don't want to fix bugs for 0.2.